### DISCUSSION TITLE
The slime/blob faction

### DISCUSSION THREAD

**Discussion by NetSysFire** [CleverRaven/Cataclysm-DDA#62043] [Ideas]
Part of the Living World project (https://github.com/orgs/CleverRaven/projects/5).

There is currently next to no lore for this faction.

My current proposal for how they came into being:

The blob contaminated the groundwater. There are plenty of small organisms, including microscopic fungi and bacteria in the earth. In areas where there is an elevated level of blob particles (e.g due to many creatures dying and zombifying, or just "natural" causes such as rain), all those organisms blend together to form a sort of black slime. They would first be a tiny splatter of slime and trapping/digesting tiny bugs or earthworms to gain mass. The blob magic™ will eventually evolve them, forming the brain slimes and similar.

Due to their digestive abilities, they formed pits. Slowly churning through earth, these slime pits would take some time to form. It would take much longer for them to "chew" through rock since it is not really organic.
They are not an "incarnation of the blob itself" but many tiny organisms, which usually humify dead organic matter, put together in a weird paste which got more viscous over time, thus forming the slimes we all know and love.

Feedback welcome.

**Comment by Raiph-Benesz** [comment]
Aside from digesting carbons, how would the Blob and the zombies/ferals/other horrors and abominations it creates, how would it incorporate/digest materials made of wood, rubber, plastic, metal, etc.?

**Reply by MochaModifier** [reply]
Presumably through adaptations or mutations, similar to how zombies don't stay normal zombies for very long in the world - over time slimes develop capabilities to consume more matter. If anything, that'd be plainly is advantageous as far as evolution goes.

With their origins starting at the microbial scale - they would evolve and adapt far quicker than macro organisms; and even faster still because of what they are.

This could be some intelligent design of a local slime hivemind thing, or simply blind, darwinian evolution where the fittest slimes survived and eventually produced a strain that grew to be as large and voracious as they are now.

**Comment by CozyVal** [comment]
If they're able to adapt and evolve faster and even adapt to eat inorganic matter, what's stopping them from just devouring the earth and becoming the dominant lifeform relatively quickly? Basically, what's the natural endpoint of their evolution?
